scripts/cfme/create_rhev_provider.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# See following link for more info:
+# http://manageiq.org/documentation/development/rest_api/models/providers/
+
+export CFME_IP="192.168.252.16"
+export RHEV_IP="192.168.252.13"
+export RHEV_ADMIN_NAME="admin@internal"
+export RHEV_ADMIN_PASS="dog8code"
```

Comment:
I would make this RHEV_ADMIN_PASS=$1 to let the user enter it at the command line.